### PR TITLE
feature/PEER-609-1: Add navigation buttons

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/components/Documentation/EditDocumentation.tsx [hunk 1]
```diff
@@ -142,6 +145,24 @@ const EditDocumentation: React.FC<EditDocumentationProps> = ({
     });
   };
 
+  const onClickArticle = (id: string): void => {
+    if (
+      typeof setViewArticleDispatch === 'function' &&
+      typeof setEditArticleDispatch === 'function'
```
Comment (almondy22) [inline app/components/Documentation/EditDocumentation.tsx:151]: Those variables can have other from function type? If you wanted to check it for `undefined` value, there's no need to check type of it

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+import ArrowDownIcon from 'icons/ArrowDown';
+import { ButtonPaginationProps } from '../../types';
```
Comment (almondy22) [inline app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx:5]: Bad practice, better use `components/Documentation/types`

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    <button
+      css={{
+        width: '350px',
+        height: '65px',
+        border: '1px solid rgba(53, 74, 137, 0.15);',
+        ':hover': {
+          background: 'rgba(53, 74, 137, 0.05)',
+        },
+      }}
+      className="ml12 mr12 mb12"
```
Comment (almondy22) [inline app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx:25]: Using two css libraries at the same time is bad, rewrite it using only emotion.
Also try to move styles to separate file or object

File: app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      className="full-width pa l0 b0"
+      css={{
+        height: '60px',
+        border: '1px solid rgba(53, 74, 137, 0.15)',
+        backgroundColor: 'var(--color-white)',
```
Comment (almondy22) [inline app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx:21]: Same as above

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    return treeArray.find(
+      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
+    )?.node;
+  };
+
+  const onClickPaginationArticle =
+    (typeButton: typeof PREV_TYPE_BUTTON | typeof NEXT_TYPE_BUTTON) => () => {
+      const { id, title } = getButtonContent(typeButton);
+
+      if (typeof onClickPaginationArticleEditDocumentation === 'function') {
```
Comment (almondy22) [inline app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:44]: Same check as above, do you need it?

File: app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx (hunk 4 above)
Comment (almondy22) [reply app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx:21]: Also check other places and fix

File: app/components/Documentation/helpers.tsx [hunk 6]
```diff
@@ -180,3 +182,21 @@ export const isEditableChildItem = (
   }
   return false;
 };
+
+export const getDataFromTree = (
+  documentationMenu: Array<DocumentationItemMenuType>,
+) => {
+  return getFlatDataFromTree({
+    treeData: documentationMenu?.map((node) => ({ ...node })),
```
Comment (almondy22) [inline app/components/Documentation/helpers.tsx:190]: For what this?

**Review by almondy22 — CHANGES_REQUESTED** [review]

File: app/components/Documentation/EditDocumentation.tsx (hunk 1 above)
Comment (alexeishulga) [reply app/components/Documentation/EditDocumentation.tsx:151]: It was Denis Romanov's decision. We can fix this during refactoring.

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 5 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:45]: It was Denis Romanov's decision. We can fix this during refactoring.

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx (hunk 2 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1385/commits/a7097e81a0e7d73ac85d3d914cf0d34be5df315d

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx (hunk 3 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1385/commits/a7097e81a0e7d73ac85d3d914cf0d34be5df315d

File: app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx (hunk 4 above)
Comment (alexeishulga) [reply app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1385/commits/a7097e81a0e7d73ac85d3d914cf0d34be5df315d

File: app/components/Documentation/helpers.tsx (hunk 6 above)
Comment (alexeishulga) [reply app/components/Documentation/helpers.tsx:190]: This is the code from the official documentation of the library

**Review by AweAwe95 — APPROVED** [review]

File: app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx (hunk 3 above)
Comment (almondy22) [reply app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx]: Confirm

File: app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx (hunk 4 above)
Comment (almondy22) [reply app/components/Documentation/components/Pagination/ButtonPaginationMobile.tsx]: Confirm

File: app/components/Documentation/components/Pagination/PaginationDocumentation.tsx (hunk 5 above)
Comment (almondy22) [reply app/components/Documentation/components/Pagination/PaginationDocumentation.tsx:45]: Confirm

**Review by almondy22 — APPROVED** [review]

**Pull request merged**